tests/nlu/test_persistor.py

```diff
@@ -7,51 +9,13 @@
 from rasa.nlu import persistor
 import rasa.nlu.train
 from rasa.nlu.config import RasaNLUModelConfig
+from rasa.nlu.persistor import Persistor
 
 
 class Object:
     pass
 
 
-async def test_list_method_method_in_AWS_persistor(  # noinspection PyPep8Naming
```

Comment:
why removing all of these tests?

Reply:
I removed the list methods of all persistors as it's not used anywhere. Hence, the tests are going with it.